### ISSUE TITLE
Transferring maintainership  

### ISSUE THREAD

**Issue by jariz** [iansinnott/date-fns-dash-docset#4]
Hi @iansinnott,

Would you be interested in transferring maintainership?
I recently made a PR over on the user contributions repo (Kapeli/Dash-User-Contributions#2615) that got merged, and now the 2 repo's aren't in sync anymore, which sucks.

You made this project a while ago and you're not that interested anymore, I get that, but I use dash & date-fns a lot, and I would like to have an up to date docset on there (even if it's just for my own personal use).

Otherwise, I'm probably going to make a fork and update the user contributions repo to reference the fork from now on.

**Comment by iansinnott** [comment]
> Would you be interested in transferring maintainership?

Sure

**Comment by jariz** [comment]
Thanks! Can you send the invite again, I didn't check my mail over the weekend and it expired.

**Comment by caweidmann** [comment]
Hi @jariz thanks for taking it up to update the docs! I also use them daily and it's a huge help to have up to date docs in dash.

**Comment by iansinnott** [comment]
Hm, I could have sworn I transferred this. Well, if anyone is still interested in taking over the repo I'm happy to transfer. I have indeed not used the underlying lib much since creating this docset.

**Comment by caweidmann** [comment]
@iansinnott if you can show me how to update the docs then I can take ownership. I still use the lib every now and then.

**Comment by jariz** [comment]
FYI: It's not as simple as just taking it over, the code as it is right now does not work on the latest dfns website.
My fork here should probably still work: https://github.com/jariz-forks/date-fns-dash-docset

Other than that, fine with me :)

**Comment by iansinnott** [comment]
@caweidmann yeah this code hasn't been touched in a while, I'd myself would have to read through it all again to see whats going on.

@jariz oh, nice. I'll add a link to that in the readme.

**Comment by caweidmann** [comment]
What's the worst that can happen? :D I guess I just need a primer on how this actually pulls through to Dash. If there is a release on this repo does that automatically reflect in Dash?

**Comment by iansinnott** [comment]
There's a script which, if I remember correctly, submits a PR to the Dash repo which is the source of truth for community docsets
